shared/css/color.scss

```diff
@@ -97,3 +96,16 @@ $bootstrap_error_border: #ebccd1;
 $bootstrap_warning_background: #fcf8e3;
 $bootstrap_warning_text: #c09853;
 $bootstrap_warning_border: #faebcc;
+
+// Colors taken from droplet, not part of our core theme.
+$droplet-light-green: #d3e965;
+$droplet-blue: #64b5f6;
+$droplet-bright-blue: #19c3e1;
+$droplet-yellow: #fff176;
+$droplet-orange: #ffb74d;
+$droplet-red: #f78183;
+$droplet-cyan: #4dd0e1;
+$droplet-pink: #f57ac6;
+$droplet-purple: #bb77c7;
+$droplet-green: #68d995;
+$droplet-white: $white;
```

Comment:
I just lifted them as-is; I can certainly do the rename though, that seems like a good idea.